src/unicorn_baseline/language/tasks/Task012_predicting_histopathology_sample_origin.json

```diff
@@ -0,0 +1,23 @@
+{
+    "Task": "Histopathology sample type",
+    "Type": "Classification",
+    "Description": "This task involves extracting the origin of the material described in the pathology report. The output should classify the tissue origin into one of the following categories: lung, lymph node, bronchus, liver, brain, bone, or other. The origin of the tissue is generally mentioned at the beginning of the report as aard materiaal.",
+    "Data_Path": "test.json",
+    "Input_Field": "text",
+    "Label_Field": "single_label_multi_class_classification_target",
```

Comment:
This is a repetition from `/input/nlp-task-configuration.json -> label_name`, please use that value to prevent any accidental inconsistency errors.